src/containers/offer-page/specialization-block/SpecializationBlock.tsx

```diff
@@ -45,7 +48,19 @@ const SpecializationBlock = <T extends CreateOrUpdateOfferData>({
   const handleCheckboxesChange = (value: ProficiencyLevelEnum[]) => {
     handleNonInputValueChange('proficiencyLevel', value)
   }
-  const levelOptions = Object.values(ProficiencyLevelEnum)
+
+  const levelLabels = useMemo(
+    () =>
+      new Map(
+        Object.entries(proficiencyLevelLabels).map(([key, value]) => [
+          key,
+          t(value)
+        ])
+      ),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You are doing smth wrong if you need this in order to code work properly.

Reply:
ESLint says to add `t` into dependencies, but I initially thought that since the values in `proficiencyLevelLabels` are constants and won't change, it wasn't necessary. However, after researching, I agree with you. I still need to include `t` in the dependency array to ensure the memoized value is recomputed if necessary. Thanks for the suggestion. I'll make the change right now